src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -504,6 +506,12 @@ class VisitService(
     }
   }
 
+  private fun handleVisitUpdateEvents(existingBooking: Visit, updatedVisit: Visit) {
+    if (existingBooking.visitStart.toLocalDate() != updatedVisit.visitStart.toLocalDate()) {
+      visitNotificationEventRepository.deleteByBookingReferenceAndType(existingBooking.reference, NotificationEventType.PRISON_VISITS_BLOCKED_FOR_DATE)
+    }
```

Comment:
I suppose just your just doing PRISON_VISITS_BLOCKED_FOR_DATE for now

Reply:
yes the other ones do not change with a visit date change. we might have to revisit this for non associations.